packages/content/config/services.xml

```diff
@@ -101,7 +101,9 @@
             <argument type="service" id="sulu_content.content_structure_bridge_factory"/>
             <argument type="service" id="sulu_http_cache.cache_lifetime.resolver"/>
 
-            <tag name="sulu_route.defaults_provider"/>
+            <tag name="sulu_route.route_defaults_provider" resource-key="examples"/>
```

Comment:
I guess this is only a temporary solution?

Reply:
@Prokyonn yes, we need kind of find out which `doctrine` entities exist at compile / compiler pass where doctrine service not exists yet. I couldn't find a quick way for this and decided for the alphe with this workaround.